docs/data/data-grid/custom-columns/EditingWithDatePickers.tsx

```diff
@@ -81,7 +65,20 @@ function GridEditDateCell({
       value={value}
       autoFocus
       onChange={handleChange}
-      slots={{ textField: WrappedGridEditDateInput }}
+      slotProps={{
```

Comment:
Nit: Is there a way to force the "shrink" state for the floating label on pickers? If so, would be good to add it for consistency.

Picker field:
<img width="576" alt="Screenshot 2024-12-24 at 13 40 27" src="https://github.com/user-attachments/assets/b1219053-0ad5-47b7-98da-54d60f20bd04" />

Text field:
<img width="575" alt="Screenshot 2024-12-24 at 13 40 15" src="https://github.com/user-attachments/assets/8fc3fa3b-b0c5-4452-b81a-564e8426d5e8" />

Reply:
Sadly, Pickers with `accessibleFieldDOMStructure` do not support `placeholder` prop just like the [native date input ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date) or [React Spectrum fields](https://react-spectrum.adobe.com/react-aria/DateField.html).
It's not possible, because there is no `input` element. 🙈